.github/workflows/CI.yml

```diff
@@ -17,6 +18,18 @@ jobs:
           - beta
           - nightly
           - "1.48.0"
+        triple:
+          - { name: "atk", features: "v2_34", nightly: "--all-features", test_sys: true }
+          - { name: "cairo", features: "png,pdf,svg,ps,use_glib,v1_16,freetype,script,xcb,xlib,win32-surface", nightly: "--features 'png,pdf,svg,ps,use_glib,v1_16,freetype,script,xcb,xlib,win32-surface'", test_sys: true }
+          - { name: "gdk", features: "v3_24", nightly: "--features v3_24", test_sys: true }
+          - { name: "gdk-pixbuf", features: "v2_40", nightly: "--all-features", test_sys: true }
+          - { name: "gdkx11", features: "v3_24", nightly: "--all-features", test_sys: true }
+          - { name: "gio", features: "v2_66", nightly: "--all-features", test_sys: false }
+          - { name: "glib", features: "v2_66", nightly: "--all-features", test_sys: true }
+          - { name: "graphene", features: "v1_10", nightly: "", test_sys: false }
+          - { name: "gtk", features: "v3_24_9", nightly: "--all-features", test_sys: true }
+          - { name: "pango", features: "v1_46", nightly: "--all-features", test_sys: false }
+          - { name: "pangocairo", features: "", nightly: "--all-features", test_sys: true }
```

Comment:
having features as the empty string failed for me because cargo does not like --features with no param... I went with having "args" (including the `--features` part) in the table, this also lets you handle the examples